doc/api/training/smd_data_parallel_release_notes/smd_data_parallel_change_log.rst

```diff
@@ -7,9 +7,47 @@ Release Notes
 New features, bug fixes, and improvements are regularly made to the SageMaker
 distributed data parallel library.
 
-SageMaker Distributed Data Parallel 1.5.0 Release Notes
+SageMaker Distributed Data Parallel 1.6.0 Release Notes
 =======================================================
 
+*Date: Dec. 15. 2022*
+
+**New Features**
+
+* New SMDDP Collectives support for the SageMaker model parallelism library’s sharded data parallelism operating AllGather. For more information, see `Sharded data parallelism with SMDDP Collectives <https://docs.aws.amazon.com/sagemaker/latest/dg/model-parallel-extended-features-pytorch-sharded-data-parallelism.html#model-parallel-extended-features-pytorch-sharded-data-parallelism-smddp-collectives>`_ in the Amazon SageMaker Developer Guide.
+* Added support for Amazon EC2 ml.p4de.24xlarge instances.
+
+**Improvements**
+
+* Improved general performance of the SMDDP AllReduce collective communication operation.
```

Comment:
wanna check the wording - "general performance" vs "general improvements" is different, @Arjunbala?

Reply:
Arjun first suggested "General performance improvements to SMDDP All-Reduce collective". Let me know if we just want to keep it as he suggested.

Reply:
I'm going to just directly use Arjun's suggestion right now.

Reply:
It's both, so we're good :)